README.md

```diff
@@ -40,10 +40,14 @@ Jump To:
    [Installation section of the User Guide][docs-installation] for more
    detailed information about installing the SDK through Composer and other
    means.
-1. **Using the SDK** – The best way to become familiar with how to use the SDK
+4. **Using the SDK** – The best way to become familiar with how to use the SDK
    is to read the [User Guide][docs-guide]. The
    [Getting Started Guide][docs-quickstart] will help you become familiar with
    the basic concepts.
+5. **Beta: Removing unused services** — To date, there are over 300 AWS services available for use with this SDK.
+   You will likely not need them all. If you use Composer and would like to learn more about this feature,
+    please read the [linked documentation]().
```

Comment:
Thanks for pointing this out— I was leaving this empty because there's technically not a working link in the repo for the README, but I'll add the expected url.